components/groups.js

```diff
@@ -286,6 +286,64 @@ SteamCommunity.prototype.scheduleGroupEvent = function(gid, name, type, descript
 	}, "steamcommunity");
 };
 
+SteamCommunity.prototype.editGroupEvent = function (gid, id, name, type, description, time, server, callback) {
+	if (typeof gid === 'string') {
+		gid = new SteamID(gid);
+	}
+
+	// Event types: ChatEvent - Chat, OtherEvent - A lil somethin somethin, PartyEvent - Party!, MeetingEvent - Important meeting, SpecialCauseEvent - Special cause (charity ball?), MusicAndArtsEvent - Music or Art type thing, SportsEvent - Sporting endeavor, TripEvent - Out of town excursion
+	// Passing a number for type will make it a game event for that appid
+
+	if (typeof server === 'function') {
+		callback = server;
+		server = {"ip": "", "password": ""};
+	} else if (typeof server === 'string') {
+		server = {"ip": server, "password": ""};
+	} else {
+		server = {"ip": "", "password": ""};
```

Comment:
I realize this is copied from `scheduleGroupEvent` (where it also appears to be incorrect), but doesn't this effectively break the `{"ip": "123.123.123.123", "password": "foobar"}` usage for `server`?